when-to-cloud.qmd

```diff
@@ -0,0 +1,42 @@
+---
+title: "When To Cloud"
+subtitle: "Is cloud access and analysis for you?" 
+date: last-modified
+author: "NASA Openscapes Team"
+citation_url: https://nasa-openscapes.github.io/earthdata-cloud-cookbook/when-to-cloud.html
+slug: index
+---
+
+## When To Cloud
+
+- opening sentence(s) with link to homepage/Welcome on cloud
+
+### Questions to Ask Yourself
+- what does it mean to be in the cloud?
+        - accessing services via internet? no
+        - logging into an AWS/Azure/etc account (awareness)? yes    
+- include workflow type figures and discuss
+    - link to any relevant tutorials that demo these workflows
+    - how can you "cloud"?
+    - what's already available (services that assist)
+- listing questions from Andy's slides
+    - a bit of discussion + examples for each question
+    - concrete examples accomapnying each question
+    - "I have a TB of storage on my laptop...." storage scenarios
+    - (see this from Ryan Abernathy: https://medium.com/pangeo/closed-platforms-vs-open-architectures-for-cloud-native-earth-system-analytics-1ad88708ebb6)
+    - include a question on actions that are already services (subsetting, etc.)
+
+
+### Challenges
+It is important to be aware of the drawbacks and challenges associated with working in the cloud. Common feedback from early cloud adopters are summarized here:
+    - feedback from Champions/workshop participants
+    - (see Alexis ESIp poster)
+    - (this could be its own section on website?)
+
+### Considerations
+We are now accustomed to living in a highly digital world, separated from physical reminders of the services we use. No longer do we access documents from a row of filing cabinets, we now store them in cloud-based archives (e.g. Google Docs). We run code on high-performance computing clusters, removed from the whirring and warmth generated by servers that live away from our desks. The following are considerations for using cloud-based resources:
```

Comment:
```suggestion
We are now accustomed to living in a highly digital world, separated from physical reminders of the services we use. No longer do we access documents from a row of filing cabinets, we now store them in cloud-based archives (e.g. Google Docs). We run code on high-performance computing clusters, removed from the whirring and warmth generated by servers that live away from our desks. The following are considerations for using cloud-based resources:

```

Reply:
I've added a hard return after text and before bullet list, thinking that is required for bullets to appear, not folded into a paragraph